### DISCUSSION TITLE
Can you use Coqui to inference a multilingual multispeaker vits-fast-fine-tuning model?

### DISCUSSION THREAD

**Discussion by DogeLord081** [coqui-ai/TTS#2928] [General Q&A] [answered]
https://github.com/Plachtaa/VITS-fast-fine-tuning This is the repo that I trained the voice with. The trained voice comes with a G_latest.pth, a finetune_speaker.json, and a moegoe_config.json.

**Comment by manmay-nakhashi** [comment, marked as answer]
Other repository's models can't be clubbed with this repo , you can fine-tune in this repo and then use that model.
